content/docs/command-reference/pull.md

```diff
@@ -117,6 +117,9 @@ used to see what files `dvc pull` would download.
   `dvc repro <stage_name>` is necessary to checkout these files (into the
   workspace) and update `dvc.lock`.
 
+- `--allow-missing` - allows the command to succeed even if some files or
+  directories are missing.
```

Comment:
It might be worth including why it's useful here (for automated workflows like CI/CML).